octoprint_navbartemp/templates/navbartemp_navbar.jinja2

```diff
@@ -1,5 +1,6 @@
 <div class="navbar-text">
-    <span id="socInfo" data-bind="attr: { title: 'System On Chip Temperature' }, html: socTemp() + custCmd(), visible: hasSoc"></span>
+    <span id="socInfo" data-bind="attr: { title: 'System On Chip Temperature' }, html: socTemp(), visible: hasSoc"></span>
+	<span id="Custom" data-bind="attr: { title: 'Custom Temperature' }, html: custCmd(), visible: hasSoc"></span>
```

Comment:
do you think that this should be dependent on hasSoc value?

Reply:
tbh i am beginner so was unsure what else to attach(don't know what options are there) it to since the pi is pretty much always there.

Reply:
btw custom command i not always custom temperature. Try to change names. Main purpose to custom command is to add additional information, temperature, cpu load, etc. and is independent of mechanism which is checking SoC.

Reply:
Wel sure I am open for suggestions how to do it now its a local hack i applied love to make it usefull for you but I could use a bit of guidance.

Reply:
tbh have no idea ;) I also tried to develop this plugin but don't know anything about jinja2 so I'm open for suggestions. Also now a days I don't use octoprint.